roles/satellite-clone/tasks/ensure_pulp_data_permissions.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: 'Ensure pulp user exists'
```

Comment:
Does this also ensure the group exists? I don't know the Ansible `user` resource, but later on it's assumed the group exists. So either the description is incorrect or you're missing a step.

Reply:
It uses adduser and that by default does create the group.

But I could also create the group explicitly.

Reply:
(done)